### PR TITLE
[LLM] Example for deploy Mixtral model

### PR DESCRIPTION
**Pull request by Michaelvll**
This PR adds the example for deploying Mixtral model with both normal SkyPilot cluster and SkyServe.

Tested (run the relevant ones):

- [ ] Code formatting: `bash format.sh`
- [x] Any manual or new tests for this PR (please specify below)
  - [x] `sky launch -c mixtral serve.yaml`
  - [x] `sky serve up -n mixtral serve.yaml`
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `bash tests/backward_comaptibility_tests.sh`

### CODE REVIEW

File: llm/mixtral/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+resources:
+  accelerators: {A100:4, A100:8, A100-80GB:2, A100-80GB:4, A100-80GB:8}
+```
+
+## 2. Serve with multiple instances
+
+When scaling up is required, SkyServe is the library built on top of SkyPilot, which can help you scale up the serving with multiple instances, while still providing a single endpoint. To serve Mixtral with multiple instances, run the following command:
+
+```bash
+sky serve up -c mixtral ./serve.yaml
```
Comment (concretevitamin) [inline llm/mixtral/README.md:25]:
```suggestion
sky serve up -n mixtral ./serve.yaml
```

File: llm/mixtral/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    post_data:
+      model: mistralai/Mixtral-8x7B-Instruct-v0.1
+      messages:
+        - role: user
+          content: Hello! What is your name?
+    initial_delay_seconds: 1200
+  replica_policy:
+    min_replicas: 1
+    auto_restart: true
+```
```
Comment (concretevitamin) [inline llm/mixtral/README.md:42]: Maybe add a short para here "Optional: Using spot instance replicas to save 3-4x cost"?

File: llm/mixtral/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Serving Mixtral from Mistral.ai
+
+Mistral AI released Mixtral 8x7B, a high-quality sparse mixture of experts model (SMoE) with open weights. Mixtral outperforms Llama 2 70B on most benchmarks with 6x faster inference. Mistral.ai uses SkyPilot as [the default way](https://docs.mistral.ai/self-deployment/skypilot) to distribute their new model. This folder contains the code to serve Mixtral on any cloud with SkyPilot. There three ways to serve the model:
```
Comment (concretevitamin) [inline llm/mixtral/README.md:3]:
```suggestion
Mistral AI released Mixtral 8x7B, a high-quality sparse mixture of experts model (SMoE) with open weights. Mixtral outperforms Llama 2 70B on most benchmarks with 6x faster inference. Mistral.ai uses SkyPilot as [the default way](https://docs.mistral.ai/self-deployment/skypilot) to distribute their new model. This folder contains the code to serve Mixtral on any cloud with SkyPilot. 

There are three ways to serve the model:
```

**Review by concretevitamin — COMMENTED** [review]
Awesome @Michaelvll! Some nits.

File: llm/mixtral/serve.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
+service:
```
[the remark this answers is not in the document]
Comment (Michaelvll) [reply llm/mixtral/serve.yaml:6]: Done. PTAL.

File: llm/mixtral/README.md (hunk 2 above)
Comment (Michaelvll) [reply llm/mixtral/README.md:60]: Good call. Added.

File: llm/mixtral/README.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+  -d '{
+      "model": "mistralai/Mistral-7B-v0.1",
+      "prompt": "My favourite condiment is",
+      "max_tokens": 25
+  }'
+```
+
+## 2. Serve with multiple instances
+
+When scaling up is required, SkyServe is the library built on top of SkyPilot, which can help you scale up the serving with multiple instances, while still providing a single endpoint. To serve Mixtral with multiple instances, run the following command:
```
Comment (concretevitamin) [inline llm/mixtral/README.md:40]: Nit: link to SkyServe docs

File: README.md [hunk 6]
```diff
@@ -27,6 +27,7 @@
 
 ----
 :fire: *News* :fire:
+- [Dec, 2023] [**Mixtral 8x7B**](https://mistral.ai/news/mixtral-of-experts/), a high quality sparse mixture-of-experts model, was released by Mistral AI! Deploy via SkyPilot on any cloud: [**official Mistral docs**](https://docs.mistral.ai/cloud-deployment/skypilot/) (may not work yet, due to the outdated docker image), [**our SkyServe Example**](./llm/mixtral/).
```
Comment (concretevitamin) [inline README.md:30]:
```suggestion
- [Dec, 2023] [**Mixtral 8x7B**](https://mistral.ai/news/mixtral-of-experts/), a high quality sparse mixture-of-experts model, was released by Mistral AI! Deploy via SkyPilot on any cloud: [**single replica**](https://docs.mistral.ai/cloud-deployment/skypilot/); [**multiple replicas**](./llm/mixtral/).
```
How does this look?

**Review by concretevitamin — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (Michaelvll) [reply README.md]: Sounds good! Changed.

**Pull request merged**